about.html

```diff
@@ -0,0 +1,132 @@
+          </ul>
+        </nav>
+        <a href="#campaign" class="campaign campaign-link tetiary-font">CC Campaign</a>
+      </nav>
+      
+     
+    </div>
+  
+    <div class="about-header-body">
+      <p class="about-headline-header primary-font">"Hello! Sharing world"</p>
+      <p class="about-headline-body-about secondary-font">CREATIVE WRITERS GLOBAL SUMMIT 2015</p>
+    </div>
+    <div class="headline-body-intro">
+      <p class="about-body-intro primary-font">The Global Summit brings together the community of writters in the  financial worlds, Intellectual, academics,  and activists who comprise the Creative Writers affiliate network in a different country every two years.
+        This year, we hope to expand our invitation list by including organizations and individuals who want to work with us on shared projects that advance the cause of the in every espect of life
+      </p>
+    </div>
+    
+    <div class="contact-submit secondary-font">
+      <p class="contact-body-intro primary-font">Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+      <a href="ccsummit2015@ccNigeria.org" class="about-connect primary-font">ccsummit2015@ccNigeria.org</a>
+      <P ></P>
+    </div>
+  </header>
+  <section class="logo-program-section">
+    <div class="logo-section">
+      <div class="logo-creation">
+       <p class="logo-creation-title tetiary-font">Creative Writers 
+        Global Summit 2015 Logo</p> 
+        <p class="logo-creation-body primary-font">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+        </p>
+      </div>
+      
+    </div>
+    <img src="./assets/images/logo.svg" alt="" class="logoImg">
+  </section>
+  <section class="past-program">
+    <div class="past-section">
+      <h1 class="past-program-header tetiary-font">See the past CC Global Summits</h1>
+      <p class="past-program-body primary-font">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+    </div>
+    <div class="pastProgram">
+      <div href="" class="past-program1">
+        <div class="overlaypast"></div>
+        <a href="#" class="past-link">
+          <h1 class="text tetiary-font">2015</h1>
+          <p class="text1 primary-font">CC Global Submit 2014</p>
+        </a>
+      </div>
+      <div href="" class="past-program2">
+        <div class="overlaypast"></div>
+        <a href="#" class="past-link">
+          <h1 class="text tetiary-font">2019</h1>
+          <p class="text1 primary-font">CC Global Submit 2019</p>
+        </a>
+      </div>
+    </div>
+  </section>
+
+  <footer id="partners">
+    <div class="partner-section">
+      <h1 class="footer-title tetiary-font">Partner</h1>
+      <div class="partner-link">
+        <a href="" class="mozilla"></a>
+        <a href="" class="google"></a>
+        <a href="" class="naver"></a>
```

Comment:
- [ ]  Previous reviewer suggested you to reduce the space between patrner's logo, so that you can resemble your design to the one from behance design, but seemed like you've not changed it. Please arrange those logos very well to match the provided design. 
- [ ] Previous reviewer suggested to underline the partners text with an orange line as it is provided from the design, but you've not implemented that as well. 
![image](https://user-images.githubusercontent.com/43877689/227729918-7dc2f1b3-07f8-4e7d-bae3-282140fab7cc.png)